packages/amplication-server/src/core/assistant/assistant.service.ts

```diff
@@ -665,5 +674,58 @@ export class AssistantService {
             : "Entity",
       }));
     },
+    getPlugins: async (args: undefined, context: AssistantContext) => {
+      const plugins = (await this.pluginCatalogService.getPlugins()).map(
+        (plugin) => omit(plugin, ["__typename"])
```

Comment:
keep the data manipulation close to the action. 
`PluginCatalogItem` doesn't contain `__typename` so this line should have not effect. A unit tests would have caught it.